testing/scenario_app/lib/src/scenarios.dart

```diff
@@ -46,6 +46,7 @@ Map<String, ScenarioFactory> _scenarios = <String, ScenarioFactory>{
   'initial_route_reply': () => InitialRouteReply(PlatformDispatcher.instance),
   'platform_view_with_continuous_texture': () => PlatformViewWithContinuousTexture(PlatformDispatcher.instance, 'Platform View', id: _viewId++),
   'bogus_font_text': () => BogusFontText(PlatformDispatcher.instance),
+  'spawn_engine_works' : () => BogusFontText(PlatformDispatcher.instance),
```

Comment:
You're compounding 2 tests. Maybe something generic like PoppableScreenScenario?

Reply:
I planned on doing this.  Unfortunately I spent hours yesterday trying to fix my scenario tests.  I keep getting a bad kernel error.  I can't figure it out so I can't work on this right now.

Reply:
https://github.com/flutter/flutter/issues/72525